integration/ledger/snapshot_test.go

```diff
@@ -1037,25 +1036,43 @@ func invokeWithoutPassingOrdererEndPoint(n *nwo.Network, peer *nwo.Peer, channel
 
 // commitTx commits a transaction for a given transaction envelope
 func commitTx(n *nwo.Network, orderer *nwo.Orderer, peer *nwo.Peer, channelID string, env *cb.Envelope, txid string) error {
-	var cancelFunc context.CancelFunc
-	ctx, cancelFunc := context.WithTimeout(context.Background(), 10*time.Second)
-	defer cancelFunc()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
This configures the timeout on the `DeliverFiltered` service. The context will be canceled when if the timeout occurs `Recv` will return with a `context.Canceled` error that is percolated out.

Reply:
Great, thanks.